client/galaxy/scripts/entry/analysis/AnalysisRouter.js

```diff
@@ -339,24 +312,11 @@ export const getAnalysisRouter = (Galaxy) =>
         },
 
         show_workflows_import: function () {
-            const workflowImportInstance = Vue.extend(WorkflowImport);
-            const vm = document.createElement("div");
-            this.page.display(vm);
-            new workflowImportInstance().$mount(vm);
+            this._display_vue_helper(WorkflowImport);
         },
 
         show_custom_builds: function () {
-            const historyPanel = this.page.historyPanel.historyView;
-            if (!historyPanel || !historyPanel.model || !historyPanel.model.id) {
-                window.setTimeout(() => {
```

Comment:
Worth noting that this does actually change the behavior here, though I don't recall why we had to have a delay in.

Reply:
Ahh.  You can see the problem this was there to address by navigating directly to `http://galaxy_app/custom_builds`.  Central panel loads before the history, causing an issue.

The workaround that was in place isn't great -- the component itself should probably recheck.  Would you like to address that here so there's not a (admittedly minor) regression?

Reply:
Hey Dannon, I took your advice and moved the check history function from the analysis router to the custom builds component. This should migrate the issue.

Reply:
@lawrence14701 Thanks!